### ISSUE TITLE
Move to case insensitive collation for frequently searched columns

### ISSUE THREAD

**Issue by Brutus5000** [FAForever/db#177]
Some columns are frequently searched via our API. Users usually do not want to search case sensitive. This is however the most used collation in our db. Performan lower(...) on a column for search is a huge performance issue. The main goal is therefore to move all (or at least all relevant) columns to case insensitive collation.

**Important note:** For unique columns, it needs to be enforced first that there are no collisions.

**Relevant table columns:**
- [ ] login.login
- [ ] login.email
- [ ] clan.name
- [ ] clan.tag
- [ ] map.display_name
- [ ] mod.display_name
- [ ] game_stats.gameName

**Comment by Brutus5000** [comment]
**Current investigation results:**
Duplicate usernames (case insensitive): 19
Duplicate emails (case insensitive): 510

To find collisions in login tables the following queries can be used:

```
DROP TEMPORARY TABLE login_names;
DROP TEMPORARY TABLE login_names2;

CREATE TEMPORARY TABLE login_names
(PRIMARY KEY id_pkey(id), INDEX lowercase_name (lower_login), INDEX lowercase_email(lower_email))
SELECT id, login, email, lower(login) as lower_login, lower(email) as lower_email FROM login;

CREATE TEMPORARY TABLE login_names2
(PRIMARY KEY id_pkey(id), INDEX lowercase_name (lower_login), INDEX lowercase_email(lower_email))
SELECT id, login, email, lower(login) as lower_login, lower(email) as lower_email FROM login;

select L1.id, L1.login, L1.email, L2.id as "OverlapId", L2.login as "OverlapsWithLogin", L2.email
from login_names L1
inner join login_names2 L2 ON L1.id != L2.id and L1.lower_login = L2.lower_login
order by L1.lower_login;

select L1.id, L1.login, L1.email, L2.id as "OverlapId", L2.login, L2.email as "OverlapsWithEmail"
from login_names L1
inner join login_names2 L2 ON L1.id != L2.id and L1.lower_email = L2.lower_email
order by L1.lower_email;
```

**Comment by Brutus5000** [comment]
Fixed with 6fdd29ef600905092b497613c7b8c4f5b441524a
